packages/devtools_app/lib/src/inspector/inspector_tree_controller.dart

```diff
@@ -594,6 +593,13 @@ class InspectorTreeController extends Object
   }
 
   /* Search support */
+  @override
+  set search(String value) {
+    // Expand tree so all rows are available for search
+    _expandAll(root);
```

Comment:
instead of overriding search to expand the tree. Why don't we create a copy of the tree at the time of tree creation. This copy can be fully expanded from the start, which has the advantages that A) we won't have to modify the expand/collapse state of the original tree, and B) we won't have to pay the cost of expanding the tree each time we perform a search.

Reply:
Having a copy leads to a previous issue when actual\cached rows and rows from the copy are not the same instances so `isSearchMatch` doesn't reflect the actual value. Anyway I tried using a copy of the tree for search and finding a path to the root for breadcrumbs. `isSearchMatch` was made to show the actual value because `getCachedRow()` sets the value to the cached row before returning it.